vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -53,6 +53,38 @@ struct json_type_traits<react::uwp::Selection> {
   }
 };
 
+static const std::unordered_map<std::string, winrt::InputScopeNameValue>
+    textBoxKeyboardTypeMap = {
+        {"default", winrt::InputScopeNameValue::Default},
+        {"number-pad", winrt::InputScopeNameValue::TelephoneNumber},
+        {"decimal-pad", winrt::InputScopeNameValue::Digits},
+        {"email-address", winrt::InputScopeNameValue::EmailNameOrAddress},
+        {"phone-pad", winrt::InputScopeNameValue::TelephoneNumber},
+        {"numeric", winrt::InputScopeNameValue::Number},
+        {"url", winrt::InputScopeNameValue::Url},
+        {"web-search", winrt::InputScopeNameValue::Search}};
+
+static const std::unordered_map<std::string, winrt::InputScopeNameValue>
+    passwordBoxKeyboardTypeMap = {
```

Comment:
>passwordBoxKeyboardTypeMap [](start = 4, length = 26)

Can you still add the mapping of "number-pad" and other options for passwordbox?

Reply:
PasswordBox only support two input scope, IS_Password and IS_NumericPin. Setting other input scope will trigger runtime error.

---
In reply to: [325880745](https://github.com/microsoft/react-native-windows/pull/3164#discussion_r325880745) [](ancestors = 325880745)